### PR TITLE
Minor changes to improve MooseDocs presentations

### PR DESCRIPTION
**Pull request by aeslaughter**

### CODE REVIEW

File: python/MooseDocs/extensions/floats.py [hunk 1]
```diff
@@ -148,6 +148,12 @@ class RenderFloat(components.RenderComponent):
     def createHTML(self, parent, token, page): #pylint: disable=no-self-use
         div = html.Tag(parent, 'div', token)
         div.addClass('moose-float-div')
+
+        if token['bottom']:
+            cap = token(0)
+            cap.parent = None
+            cap.parent = token
```
Comment (milljm) [inline python/MooseDocs/extensions/floats.py]: Is this necessary?

**Review by milljm — COMMENTED** [review]
Just curious if you meant to set the same variable immediately to a different value.

File: python/MooseDocs/extensions/floats.py (hunk 1 above)
Comment (aeslaughter) [reply python/MooseDocs/extensions/floats.py]: Probably not, I will fix it.

**Review by milljm — DISMISSED** [review]
👍

File: python/MooseDocs/extensions/floats.py [hunk 2]
```diff
@@ -148,6 +148,12 @@ class RenderFloat(components.RenderComponent):
     def createHTML(self, parent, token, page): #pylint: disable=no-self-use
         div = html.Tag(parent, 'div', token)
         div.addClass('moose-float-div')
+
+        if token['bottom']:
+            cap = token(0)
+            cap.parent = None
```
Comment (aeslaughter) [inline python/MooseDocs/extensions/floats.py]:
```suggestion
```

File: python/MooseDocs/extensions/floats.py [hunk 3]
```diff
@@ -148,6 +148,11 @@ class RenderFloat(components.RenderComponent):
     def createHTML(self, parent, token, page): #pylint: disable=no-self-use
         div = html.Tag(parent, 'div', token)
         div.addClass('moose-float-div')
+
+        if token['bottom']:
+            cap = token(0)
+            cap.parent = token
```
Comment (aeslaughter) [inline python/MooseDocs/extensions/floats.py]:
```suggestion
            cap.parent = None # Guarantees that "cap" is removed from the current tree
            cap.parent = token
```

Comment (milljm) [reply python/MooseDocs/extensions/floats.py]: 👍

**Review by milljm — APPROVED** [review]
👍

**Pull request merged**